### DISCUSSION TITLE
NerdVille - A console Build City game

### DISCUSSION THREAD

**Discussion by OpenCode** [Textualize/textual#243] [Show and tell]
Hi. I'm creating a console game with Textual. It's a build city. It's not simple to use Textual without documentation and I'm using the library code itself to try to understand.

Here you can follow the actual development: https://github.com/OpenCode/NerdVille

Feel free to suggest ideas or issues.

**Comment by erwingame** [comment]
Interesting!

**Comment by Abdur-rahmaanJ** [comment]
Quite amazing!!!

**Comment by willmcgugan** [comment]
Yeah, wow!
